packages/nextjs/app/page.tsx

```diff
@@ -1,52 +1,193 @@
+        />
+      </div>
+
+      <div className="absolute w-full h-full pointer-events-none">
+        <Image src={blur_blue} alt="Blurred Blue Background" className="absolute -bottom-[10%] -right-[40%] blur-2xl" />
+      </div>
+
+      <div className="max-w-7xl mx-auto flex flex-col items-center justify-center">
+        {/* Punks BG */}
+        <div className="overflow-clip">
+          <Image src={punks} alt="punks" />
+        </div>
+
+        {/* Header / SubHeader */}
+        <div className="mt-10 sm:mt-16 tracking-wider flex flex-col items-center justify-center">
+          <h1 className="font-semibold text-center text-4xl md:text-5xl lg:text-6xl xl:text-7xl">
+            Welcome to Batch 9 of
+            <span className="ml-3 font-bold">
+              <span className="text-st_cyan">Buidl</span>
+              <span className="text-st_yellow">Guidl</span>
+            </span>
           </h1>
-          <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
-          <p className="text-lg flex gap-2 justify-center">
-            <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+          <p className="text-lg md:text-xl text-center lg:text-2xl xl:text-3xl">
+            Get started by taking a look at your batch GitHub repository.
           </p>
         </div>
 
-        <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
-          <div className="flex justify-center items-center gap-12 flex-col sm:flex-row">
-            <div className="flex flex-col bg-base-100 px-10 py-10 text-center items-center max-w-xs rounded-3xl">
-              <BugAntIcon className="h-8 w-8 fill-secondary" />
-              <p>
+        {/* Member Status */}
+        <div className="w-[16rem] sm:w-[34rem] text-center tracking-widest py-5 mt-6 text-st_cyan font-semibold rounded-xl border border-zinc-500 uppercase shadow-sm hover:bg-zinc-300 dark:hover:bg-zinc-800 transition-all">
+          Batch Member {isAllowed ? "✅" : "❌"}
+        </div>
+
+        {isAllowed && IscheckedIn !== zeroAddress && (
+          <div className="text-center italic text-zinc-400">
+            <p className="">Cheers 🍻.. You are checked in!</p>
+          </div>
+        )}
+
+        {/* Cards */}
+        <div className="flex flex-col sm:flex-row sm:gap-8">
+          {/* Debug Contract Card */}
+          <div className="shadow-sm border rounded-xl w-[16rem] h-[12rem] font-light mt-6 border-zinc-500 flex flex-col">
+            <div className="w-full flex items-center justify-center flex-grow-[5] dark:hover:bg-zinc-800 hover:bg-zinc-300 transition-all rounded-t-xl">
+              <p className="px-4">
                 Tinker with your smart contract using the{" "}
-                <Link href="/debug" passHref className="link">
+                <Link href="/debug" target="_blank" passHref className="link transition-all">
```

Comment:
Do we want to have the same link twice, or is one link at the bottom sufficient?

Reply:
yeah you're right. i'll just keep the bottom link. i'm anyway creating a reusable component for it so it will fit nicely